src/commands/wait-in-queue.yml

```diff
@@ -147,6 +147,7 @@ steps:
             # in the correct order.
             if [[ -n "$CHECK_PREVIOUS_COMMIT" ]]; then
```

Comment:
This condition doesn't appear to be used often so I didn't add any additional logic to guard against race conditions. This seems like a rare event and we can address it if/when it occurs and causes issue